src/frontend/web_application/src/components/MultidimensionalPi/components/PiGraph/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+import './style.scss';
+
+const Grid = ({ pi, angle, axeLength }) => {
+  const axeCoordinates = [];
+  const polygonPoints = [];
+  let count = 0;
+
+  pi.map((p) => {
+    const axeName = p.name;
+    const axeX = axeLength - (axeLength * Math.sin((count * Math.PI) / 180));
+    const axeY = axeLength - (axeLength * Math.cos((count * Math.PI) / 180));
+    axeCoordinates.push({ axeName, x: axeX, y: axeY });
+    polygonPoints.push(axeX, axeY);
+    count -= angle;
+  });
+
+  return (
+    <g className="m-pi-graph__grid">
+      <polygon className="m-pi-graph__outline" points={polygonPoints.join(' ')} />
+      {axeCoordinates.map(p =>
+        <line
+          key={p.axeName}
+          className="m-pi-graph__line"
+          id={p.axeName}
+          x1={axeLength}
+          y1={axeLength}
+          x2={p.x}
+          y2={p.y}
+        />
+      )}
+    </g>
+  );
+};
+
+Grid.propTypes = {
+  pi: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  angle: PropTypes.number.isRequired,
+  axeLength: PropTypes.number.isRequired,
+};
+
+const Polygon = ({ pi, angle, axeLength }) => {
+  const polygonPoints = [];
+  let count = 0;
+  pi.map((p) => {
```

Comment:
Looks like you might want to use `Array.prototype.reduce` here:

```js
const polygonPoints = pi.reduce((points, p) => {
  const piLevel = p.level;
  const pointX = axeLength - (piLevel * Math.sin((count * Math.PI) / 180));
  const pointY = axeLength - (piLevel * Math.cos((count * Math.PI) / 180));
  
  return [
    ...points,
    pointX,
    pointY,
  ];
}, []);
```

Though, to follow common practice of `<polygon>` usage, it's probably better to stick with `Array.Prototype.map`:
```js
const polygonPoints = pi.map((p) => {
  const piLevel = p.level;
  const pointX = axeLength - (piLevel * Math.sin((count * Math.PI) / 180));
  const pointY = axeLength - (piLevel * Math.cos((count * Math.PI) / 180));
  
  return `${pointX},${pointY}`
});
```

And for the rendering, in any case:
```jsx
return (
  <polygon
    className="m-pi-graph__shape"
    points={polygonPoints.join(' ')}
  />
);
```